content/en/security/cloud_siem/content_packs.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Content Packs
+kind: documentation
+disable_toc: false
+further_reading:
+- link: "/security/cloud_siem/log_detection_rules"
+  tag: "Documentation"
+  text: "Create log detection rules"
+- link: "security/cloud_siem/investigator"
+  tag: "Documentation"
+  text: "Learn more about the Investigator"
+- link: "/security/cloud_siem/investigate_security_signals"
+  tag: "Documentation"
+  text: "Investigate security signals"
+---
+
+## Overview
+
+[Cloud SIEM Content Packs][1] provide out-of-the box content for key security integrations. Depending on the integration, a Content Pack can include the following:
+
+- [Detection Rules][2] to provide comprehensive coverage of your environment
+- An interactive dashboard with detailed insights into the state of logs and security signals for the Content Pack
+- [Investigator][3], an interactive graphical interface for investigating suspicious activity by a user or resource
+- [Workflow Automation][4], to automate actions and accelerate investigation and remediation of issues
+- Configuration guides
+
+{{< whatsnext desc="Content Packs are grouped into the following categories:" >}}
+    {{< nextlink href="/security/cloud_siem/content_packs#cloud-audit-content-packs" >}}<u>Cloud Audit</u>: AWS CloudTrail, Azure Security, GCP Audit Logs, Kubernetes Audit Logs{{< /nextlink >}}
+    {{< nextlink href="/security/cloud_siem/content_packs#authentication-content-packs" >}}<u>Authentication</u>: 1Password, Auth0, JumpCloud, Okta{{< /nextlink >}}
+    {{< nextlink href="/security/cloud_siem/content_packs#collaboration-content-packs" >}}<u>Collaboration</u>: Google Workspace, Microsoft 365{{< /nextlink >}}
+    {{< nextlink href="/security/cloud_siem/content_packs#network-content-packs" >}}<u>Network</u>: Cloudflare{{< /nextlink >}}
+    {{< nextlink href="/security/cloud_siem/content_packs#cloud-developer-tools-content-packs" >}}<u>Cloud developer tools</u>: GitHub{{< /nextlink >}}
+    {{< nextlink href="/security/cloud_siem/content_packs#endpoint-content-packs" >}}<u>Endpoint</u>: CrowdStrike{{< /nextlink >}}
+{{< /whatsnext >}}
+
+## Cloud Audit Content Packs
+
+### AWS CloudTrail
+
+Monitor the security and compliance levels of your AWS operations.
+
+The [AWS CloudTrail Content Pack][5] includes:
+- Detection Rules
```

Comment:
can we link to the available detection rules?
https://docs.datadoghq.com/security/default_rules/#cloudtrail

Reply:
added links to the detection rules for all content packs.